lib/utils/logger.js

```diff
@@ -0,0 +1,14 @@
+var _ = require('lodash');
+var name = require('../../package.json').name;
+var datasetId = require('../config').datasetId;
+
+
+/**
+ * Function returning debug logger with prebuild signature
+ * @param filePath path of the file where logger will be used
+ */
+module.exports = function setupLogger(filePath) {
+  var str = filePath.substring(filePath.indexOf(datasetId)).replace(datasetId, name);
+
+  return require('debug')(_.trim(str.split('/').join(':'), '.js') + ':');
+};
```

Comment:
@nialldonnellyfh @wtrocki adding more context to this:
for dataset modules this is plug and play,
logs in list.js will print 
`fh-wfm-result:lib:client:mediator-subscribers:list: {message}`
it is bit long prefix but it will help massively with debugging or portal and mobile app